pkg/resolver/config_test.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package resolver_test
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/ethersphere/bee/pkg/resolver"
+)
+
+func TestParseConnectionStrings(t *testing.T) {
+	testCases := []struct {
+		desc       string
+		conStrings []string
+		wantCfg    []resolver.ConnectionConfig
+		wantErr    error
+	}{
+		{
+			desc: "tld too long",
```

Comment:
according to whom? is this an IETF definition? if so, please quote it. also `ffffff.....` is not a TLD, TLD is `com`, `ffff... ` is a subdomain